jobboard/tests/conftest.py

```diff
@@ -21,3 +24,27 @@ def example_student_a(example_institution_a):
                       major=Major.COMPUTER_SCIENCE, about="testing", graduation_date=date(2020, 7, 13))
     student.save()
     return student
+
+
+@pytest.fixture
+def example_recruiter():
```

Comment:
@ifireball said to keep the conftests file small, so maybe if these fixtures are only being used in the view_tests.py file it's better to put them in the file itself and not here

Reply:
I saw his comment on Eldar's PR but I think it makes more sense to put the fixtures here since it's just 2 more and we're probably not gonna put more fixtures in that file.